tests/database/odbc-w-enclave/odbc_proxy.c

```diff
@@ -0,0 +1,388 @@
+#include <stdlib.h>
```

Comment:
Why is this file called proxy?

Reply:
any suggestion for naming?

Reply:
What does it do?

Reply:
It includes all SQL operations(create/drop table/key, query(insert, query) data,  connect to database) through ODBC

Reply:
Not sure, my only bad suggestion would be odbc_helper. I'll let others make the call.

Reply:
I think  odbc_helper is a proper name.